Add query.max-run-time.hard-limit (#15115)
trinodb/trino: docs/src/main/sphinx/admin/properties-query-management.rst

time spend in a queue waiting, so essentially this is the time allowed for a
query to exist since creation.

``query.max-run-time.hard-limit``
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

* **Type:** :ref:`prop-type-duration`
* **Default value:** None

Hard limit for the ``query_max_run_time`` session property. The maximum

> allowed time for a query may be raised on a per-query basis using the

Reviewer: ```suggestion
allowed time for a query may be raised for a session using the
```

> ``query_max_run_time`` session property, but may not exceed this hard limit if

Reviewer: ```suggestion
``query_max_run_time`` session property, but cannot exceed this hard limit if
```

(or `may` -> `must`)

if it is specified.

> 

Reviewer 2: cc @colebow @mosabua for this file

Reviewer 3: Late to the party, but it looks good.
